extensions/csrf-reactive/runtime/src/main/java/io/quarkus/csrf/reactive/runtime/CsrfRequestResponseReactiveFilter.java

```diff
@@ -223,7 +239,7 @@ private boolean isCsrfTokenRequired(RoutingContext routing, CsrfReactiveConfig c
     private void createCookie(String csrfToken, RoutingContext routing, CsrfReactiveConfig config) {
 
         ServerCookie cookie = new CookieImpl(config.cookieName, csrfToken);
-        cookie.setHttpOnly(true);
+        cookie.setHttpOnly(config.cookieHttpOnly);
```

Comment:
https://cheatsheetseries.owasp.org/cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.html#defense-in-depth-techniques says we should set `SameSite` and `__Host-` prefix, any reason why we do not?

Reply:
@FroMage 

> Trying to manage SameSite attribute would duplicate what is already supported in in the individual extensions - Form, WebAuthn, OIDC and at the Vert.x HTTP configuration level.

In any case IMHO in this PR we should stay with the custom request header theme and we can keep evolving this feature with more PRs for sure